rpm/libhybris.spec

```diff
@@ -31,7 +31,6 @@ Requires: %{name}-libhardware = %{version}-%{release}
 Requires(post): /sbin/ldconfig
 Requires(postun): /sbin/ldconfig
 Provides: libEGL
-Provides: libEGL.so.1
```

Comment:
why this change?

Reply:
With this change, on OBS package information doesn't write out "Provides: libEGL.so.1" twice :)